File: oms/clojure/src/oms_rest_api_/@/specs/messages.clj
(ns oms-rest-api-///@.specs.messages
  (:require [clojure.spec.alpha :as s]
            [spec-tools.data-spec :as ds]
            )
  (:import (java.io File)))


(def messages-data
  {
   (ds/opt :type) string?
   (ds/opt :exchange_id) string?
   (ds/opt :message) string?
   })

(def messages-spec
  (ds/spec
    {:name ::messages
     :spec messages-data}))

File: oms/javascript-closure-angular/API/Client/Messages.js
goog.provide('API.Client.messages');

/**
 * @record
 */
API.Client.Messages = function() {}

/**
 * Type of message
 * @type {!string}
 * @export
 */
API.Client.Messages.prototype.type;

/**
 * Exchange name
 * @type {!string}
 * @export
 */
API.Client.Messages.prototype.exchangeId;

/**
 * Message
 * @type {!string}
 * @export
 */
API.Client.Messages.prototype.message;

